WooCommerce/Classes/Extensions/UIImage+Woo.swift

```diff
@@ -43,28 +165,64 @@ extension UIImage {
             .imageFlippedForRightToLeftLayoutDirection()
     }
 
-    /// Delete icon
+    /// Quote Image
     ///
-    static var deleteImage: UIImage {
-        let tintColor = StyleManager.wooCommerceBrandColor
-        return Gridicon.iconOfType(.crossCircle)
-            .imageWithTintColor(tintColor)!
+    static var quoteImage: UIImage {
+        return Gridicon.iconOfType(.quote)
+    }
+
+    /// Pages icon
+    ///
+    static var pagesImage: UIImage {
+        return Gridicon.iconOfType(.pages)
             .imageFlippedForRightToLeftLayoutDirection()
     }
 
-    /// More icon
+    /// Search icon
     ///
-    static var moreImage: UIImage {
-        let tintColor = StyleManager.wooCommerceBrandColor
-        return Gridicon.iconOfType(.ellipsis)
-            .imageWithTintColor(tintColor)!
+    static var searchImage: UIImage {
+        return Gridicon.iconOfType(.search)
             .imageFlippedForRightToLeftLayoutDirection()
     }
 
-    /// Jetpack Logo Image
+    /// Spam icon
     ///
-    static var jetpackLogoImage: UIImage {
-        return UIImage(named: "icon-jetpack-gray")!
+    static var spamImage: UIImage {
+        return Gridicon.iconOfType(.spam)
+    }
+
+    /// Returns a star icon with the given size and color
+    ///
+    /// - Parameters:
+    ///   - size: desired size of the resulting star icon
+    ///   - tintColor: desired tint color of the resulting icon
+    /// - Returns: a bitmap image
+    ///
+    static func starImage(size: Double, tintColor: UIColor) -> UIImage {
+        let starSize = CGSize(width: size, height: size)
+        return Gridicon.iconOfType(.star,
+                                   withSize: starSize)
+            .imageWithTintColor(tintColor)!
+    }
+
```

Comment:
Right after the `starImage`, insert the `starOutlineImage`.

`Gridicon.iconOfType(.starOutline)` then it needs to be added in `ProductDetailsViewController`.